docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -168,4 +168,6 @@ Name                                                         Description
                                                              that has been configured for the Trino coordinator.
                                                              See :doc:`/security/oauth2` for more details.
 ``disableCompression``                                       Whether compression should be enabled.
+``assumeLiteralNamesInMetadataCallsForNonConformingClients`` When enabled, the name patterns passed to ``DatabaseMetaData`` methods
```

Comment:
I have no idea what it means and what it does :)

Reply:
@sopel39  see https://github.com/trinodb/trino/pull/7438 / https://github.com/trinodb/trino/issues/7424#issuecomment-807537654

Reply:
Could you improve description here? I doubt whoever reads the doc will be able to find mentioned issues

Reply:
I tried to get the description as good as i can. Please let me know what you would want me  to include here.

Reply:
Maybe add explanation brief explanation when to enable it?

Reply:
Adding

> You can use this as a workaround for
                                                             applications that do not escape schema or table names when passing them
                                                             to `DatabaseMetaData` methods as schema or table name patterns.

from RNs

Reply:
thx